tools/cockpit.spec

```diff
@@ -523,6 +523,22 @@ getent group cockpit-wsinstance >/dev/null || groupadd -r cockpit-wsinstance
 getent passwd cockpit-wsinstance >/dev/null || useradd -r -g cockpit-wsinstance -d /nonexisting -s /sbin/nologin -c "User for cockpit-ws instances" cockpit-wsinstance
 
 %post ws
+if firewall-cmd --state; then
```

Comment:
Please add `>/dev/null 2>&1` to avoid confusing error messages if firewalld isn't installed or running.